db/migrate/20151102110431_populate_linked_items.rb

```diff
@@ -0,0 +1,8 @@
+class PopulateLinkedItems < ActiveRecord::Migration
+  def change
+    LinkSet.all.each do |link_set|
```

Comment:
It would be more efficient to use `find_each` here (http://apidock.com/rails/ActiveRecord/Batches/find_each)